### ISSUE TITLE
Validation loss , BLEU score 이상 : 데이터 중복 문제 의심

### ISSUE THREAD

**Issue by jenjenniee** [jenjenniee/NLP_Project#3]
<img src="https://github.com/user-attachments/assets/bea8e762-5aaf-433b-bf6a-581967edbd3c" width="400">
<img src="https://github.com/user-attachments/assets/21d7a8c3-7e4f-4570-8527-2bc085e1c5cc" width="400">

### 이슈 주제
 검증 부분 코드를 수정하였는데도 Validation loss와 BLEU score가 동일하게 지속됨

### Why?
Validation 데이터가 잘못 전달되었거나, 학습 데이터와 거의 동일하여 모델이 Validation에서 낮은 loss가 발생할 수 있음

**Comment by jenjenniee** [comment]
### 해결 방법
![IMG_7268](https://github.com/user-attachments/assets/4ba38c72-092a-41af-8b86-31824fe9de90)
![IMG_7267](https://github.com/user-attachments/assets/3d99fb38-4813-4645-9338-0049f9749267)
num_epochs = 2000 으로 실험을 해 본결과 validation bleu score는 0.3점 대인데 test  bleu score는 0.94가 나옴을 확인함. 
학습의 문제가 아니라 Training 데이터와 Test 데이터 간 중복이나 다른 문제가 있는지 의심됨.
